### PR TITLE
Add Toyhou.se provider

### PR DESCRIPTION
**Pull request by sarahjting**
Provider for https://toyhou.se/
Developer documentation: https://toyhou.se/~developer/oauth2

Thank you! :)

### CODE REVIEW

File: src/Toyhouse/Provider.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'TOYHOUSE';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [''];
```
Comment (atymic) [inline src/Toyhouse/Provider.php:19]: Don't need this, default is empty

File: src/Toyhouse/composer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Sarah Ting",
+            "email": "sarah.j.ting@gmail.com"
+        }
+    ],
+    "require": {
+        "php": "^7.2 || ^8.0",
```
Comment (atymic) [inline src/Toyhouse/composer.json:19]:
```suggestion
        "php": "^7.4 || ^8.0",
```

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by sarahjting** [comment]
Thank you for review! Have updated 🙇 

It's an art gallery site, similar to deviantArt but supports trading/selling art collections! It's relatively small, but I didn't see any requirements for what sort of providers you're accepting so thought I'd see if I could get it added hehe

File: src/Toyhouse/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+];
+```
+
+### Usage
+
+You should now be able to use the provider like you would regularly use Socialite (assuming you have the facade installed):
+
+```php
+return Socialite::driver('toyhouse')->redirect();
+```
```
Comment (atymic) [inline src/Toyhouse/README.md:42]: Can you add the returned fields here? See other providers for example on website.

Thanks :)

**Review by atymic — CHANGES_REQUESTED** [review]
One minor but good to go otherwise

**Comment by sarahjting** [comment]
Added! Thanks again ☺️

**Review by atymic — APPROVED** [review]

**Pull request merged**